web/src/timerender.ts

```diff
@@ -190,10 +190,13 @@ export function render_now(time: Date, today = new Date()): TimeRender {
 // Relative time rendering for use in most screens like Recent conversations.
 //
 // Current date is passed as an argument for unit testing
-export function relative_time_string_from_date(
-    last_active_date: Date,
+export function relative_time_string_from_date({
+    last_active_date,
```

Comment:
Let's rename this parameter `last_active_date`. The whole reason we split this function from the one for "last active" values is that they need different formats and the uses of this are not "last active" times for the buddy list -- they are just places where we want to display a relative time.

I think `date` would be a good name for the parameter; and I guess `current_date` for the second parameter if we need to pass that for some reason.

Reply:
I have renamed this parameter.

Regarding the trick with the current date, I think your idea of using `mockdate` is great. However, I believe it would be better to carry out this task in a separate pull request. There might be something that requires our closer attention. What do you think about it? Should we also create an issue just to ensure we don't forget about it?

Reply:
Opened https://github.com/zulip/zulip/issues/26774 to track this.